### ISSUE TITLE
Add generator for privacy data handling

### ISSUE THREAD

**Issue by IvanGrginovInf** [infinum/eightshift-boilerplate#224]
In order to be GDPR compliant, we could add a generator for a class / page that handles user data deletion on request.

Reference: https://github.com/dingo-d/woo-solo-api/blob/feature/2.0.0-update/src/Privacy/DataHandling.php

**Comment by iruzevic** [comment]
WP core has this out of the box

**Comment by dingo-d** [comment]
Yes, but the idea was to have something like `wp boilerplate create_privacy` (and some options) that would create the export and delete handlers for whatever custom private data you need to export/delete. It is a core feature but if you have any kind of custom data, coming from a custom database table, you need to add this to the export/delete functionality (like the [link](https://github.com/dingo-d/woo-solo-api/blob/develop/src/Privacy/DataHandling.php) describes).
